### PR TITLE
Closes #2 - Create initial index.html

### LINKED ISSUE #2: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (YarkoKhamar) [inline index.html]: Please add a new line.

Comment (les1223) [inline index.html]: please add new line

Comment (andriy-korchynskyy) [inline index.html]: newline

Comment (nataliapachva) [inline index.html]: Please add newline at end of file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
```
Comment (maryananovosad) [inline index.html]: please, add new line

File: index.html (hunk 1 above)
Comment (OnFireOwl) [inline index.html]: Please add new line at the end of file

Comment (TarasKataryna) [inline index.html]: Please add a new line

Comment (AlexLviv07) [reply index.html]: Add new line

File: index.html (hunk 2 above)
Comment (AlexLviv07) [inline index.html]: Please add new line

**Comment by les1223** [comment]
Please add new line

**Pull request merged**